docker-compose/README.md

```diff
@@ -60,7 +62,9 @@ Setup can be easily bootstrapped by using the `<run-script>`. In order to do so
   `<run-script>` is _run.sh_
 
 - Start using the \<run-script\>
-  - Update the _IOTEA\_PROJECT\_DIR_ variable in the _.env_  file to the location where you cloned the IoT Event Analytics repository. If you do not want to clone it manually from [https://github.com/GENIVI/iot-event-analytics](https://github.com/GENIVI/iot-event-analytics), the given _IOTEA\_PROJECT\_DIR_ directory will be created and the repository will be cloned. Make sure you specify the _IOTEA\_VERSION_ (in `run.properties`) in this case in order to check out a specific version of IoT Event Analytics.<br>
+  - Update the _IOTEA\_PROJECT\_DIR_ variable in the _.env_  file to the location of the iot-event-analytics
+    submodule. If _IOTEA\_PROJECT\_DIR_ does not exist, it will be created and the repository will be cloned. Make sure you specify the
```

Comment:
I think this could be confusing at development --> I would assume that the submodule is used and not another clone (if it is not set correctly).

Reply:
We can skip tis whole section, since we are bound to a specific version of IoT Event Analytics, where we can guarantee, that it works. Therefore I would delete these lines altogether

Reply:
Removed the section